src/components/application_manager/include/application_manager/smart_object_keys.h

```diff
@@ -216,6 +216,7 @@ extern const char* key_press_mode;
 // duplicate names from hmi_request
 extern const char* limited_character_list;
 extern const char* auto_complete_text;
+extern const char* auto_complete_list;
```

Comment:
@LuxoftAKutsan, @dtrunov Ok I would prefer to remove my new field from "duplicate names from hmi_request" however I think that this group of duplicate fields is a bit confusing, maybe I should delete the rest two fields in it too? (I can make it as a separate commit)